ping/rust/src/lib.rs

```diff
@@ -193,3 +201,11 @@ where
 
     Ok(())
 }
+
+pub fn transport_param(client: &testground::client::Client) -> String {
+    client
+        .run_parameters()
+        .test_instance_params
+        .get("transport").cloned()
```

Comment:
Pretty sure rustfmt would want that on a separate line now 🙃

Reply:
Ha, you're too fast for me @thomaseizinger 😉

Reply:
That, and rust analyzer needed a restart.